package/src/RNCSliderNativeComponent.web.tsx

```diff
@@ -145,18 +145,36 @@ const RCTSliderWebComponent = React.forwardRef(
       }
     }, [initialValue, updateValue, animatedValue]);
 
-    const onResize = () => {
-      hasBeenResized.current = true;
-    };
     React.useEffect(() => {
+      const invalidateContainerPosition = () => {
+        containerPositionInvalidated.current = true;
+      };
+      const onDocumentScroll = (e: Event) => {
+        if (
+          // Avoid all other checks if already invalidated
+          !containerPositionInvalidated.current &&
+          containerRef.current &&
+          // @ts-ignore
+          e.target.contains(containerRef.current)
+        ) {
```

Comment:
I'm not sure if linting will accept the formatting, but anyway I often prefer introducing a properly named const instead of a comment:
```suggestion
        const isAlreadyInvalidated = !containerPositionInvalidated.current;
        if (
          isAlreadyInvalidated &&
          containerRef.current &&
          // @ts-ignore
          e.target.contains(containerRef.current)
        ) {
```
It's just a suggestion - accept only if you feel like it 👍